src/lib/poracleMessage/commands/script.js

```diff
@@ -70,7 +70,7 @@ exports.run = async (client, msg, args, options) => {
 					mincp: ['min_cp', 0],
 					macxp: ['max_cp', 9000],
 					level: ['min_level', 0],
-					maxlevel: ['max_level', 40],
+					maxlevel: ['max_level', 50],
```

Comment:
I would rather go with 55.... The GM contains cpMultiplier up to 55 for ages... On reddit was a proof of 54 being available.... Even if it was a bug those could spawn on weekend as we do not know

Reply:
It was rather a suggestion, I appreciate the PR a lot... As it will help already :)